docs/plugins/repository-azure.asciidoc

```diff
@@ -64,8 +64,8 @@ cloud:
 
 `my_account1` is the default account which will be used by a repository unless you set an explicit one.
 
-You can set the timeout to use when making any single request. It can be defined globally, per account or both.
-Defaults to `5m`.
+You can set the client side timeout to use when making any single request. It can be defined globally, per account or both.
+It's not set by default which means that elasticsearch is using the default value set by the azure client.
```

Comment:
So I could link to the JavaDoc: http://azure.github.io/azure-storage-java/com/microsoft/azure/storage/RequestOptions.html#setTimeoutIntervalInMs(java.lang.Integer)?
Note that for now, this value is `null` indicating no maximum time. I guess it will remain as is so may be you're right and I should just write this in our doc???